docker-compose.yml

```diff
@@ -359,7 +359,7 @@ services:
         - SA_PASSWORD=${MSSQL_PASSWORD}
         - ACCEPT_EULA=Y
       volumes:
-        - ${DATA_PATH_HOST}/mssql:/var/opt/mssql
```

Comment:
Please revert this part.
all DB container best use the same way,

Reply:
@bestlong using ${DATA_PATH_HOST}, it cannot run if running on Mac. Should we have some comment and example here to let them know?